plutus-core/src/Language/PlutusCore/Core/Instance/Pretty/Classic.hs

```diff
@@ -17,8 +17,8 @@ import           PlutusPrelude
 import           Language.PlutusCore.Core.Instance.Pretty.Common ()
 import           Language.PlutusCore.Core.Instance.Recursive
 import           Language.PlutusCore.Core.Type
-import           Language.PlutusCore.Pretty                      (PrettyConst)
```

Comment:
I think it was probably me who did that.  I have no idea why!

We need `PrettyConst` because we have to say `uni 'Everywhere' PrettyConst` a lot.  I see that `Universe.Core` imports `PrettyConst`: would it be reasonable to re-export that via `Universe`?  That way we'd always have `PrettyConst` available when we're talking about universes (and we could do the same for other classes that we have to mention in universe constraints a lot).  Maybe that's a bad idea though: `PrettyConst` doesn't actually have anything to do with universes.

Reply:
> uni 'Everywhere' PrettyConst a lot. I see that Universe.Core imports PrettyConst

Oh, well, it really shouldn't, that module was designed to be Plutus-unaware, so that we can extract it into a library one day, but I stupidly forgot to write that as a comment in the module. We could have instances like

```haskell
instance (Closed uni, uni `Everywhere` PrettyBy config) => PrettyBy config (Some (ValueOf uni)) where
```

and replace `PrettyConst` at usage sites with `PrettyBy PrettyConfigPlc` or whatever.

Or we could simply move all of the pretty-printing stuff out of this module, which I think is a good idea, so I'm going to do that (maybe in a different PR, not sure).

Reply:
> Or we could simply move all of the pretty-printing stuff out of this module, which I think is a good idea, so I'm going to do that (maybe in a different PR, not sure).

While you're at it you might want to put `Parsable,hs` somewhere more sensible.  I just stuck it in `Language.PlutusCore`.  It's sort of the opposite of `PrettyConst` though, so maybe it belongs somewhere near it, or even in the same file?  There might be circumstances when the two classes need to share something, like strings representing literal names of constants (like `true` and `false` for example, if we wanted to use those instead of Haskell's `True` and `False`)

Reply:
Created a JIRA issue for that.